schemas/core/validation-outcome.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$id": "https://raw.githubusercontent.com/guardrails-ai/interfaces/core-schemas/schemas/core/validation-outcome.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "ValidationOutcome",
+  "description": "The output from a Guard execution.",
+  "type": "object",
+  "properties": {
+    "rawLlmOutput": {
+      "type": "string",
+      "description": "The raw, unchanged string content from the LLM call."
+    },
+    "validatedOutput": {
+      "description": "The validated, and potentially fixed, output from the LLM call after undergoing validation.",
```

Comment:
I'm starting to dislike this definition XD

Reply:
same